cryptus-app/components/market_overview/net_worth/net_worth.module.scss

```diff
@@ -0,0 +1,48 @@
+@use "/styles/variables" as v;
+
+.container {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: v.$bg-color;
+}
+.container_row {
+  flex: 1 1 auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  background-color: v.$bg-color;
+}
+
+.value {
+  font-family: "Arial Black", Gadget, sans-serif;
+  font-size: 36px;
+  color: #000000;
+  font-style: normal;
+  color: black;
+}
+
+.delta {
+  font-family: "Arial Black", Gadget, sans-serif;
+  font-size: 18px;
+  color: #000000;
+  font-style: normal;
+  color: black;
+}
```

Comment:
same

Reply:
What and where are globals ? @alexandrelafleur
can't seem to find .container in globals.css nor _variabeles.scss